### ISSUE TITLE
String.prototype.replaceAll not polyfilled

### ISSUE THREAD

**Issue by xwenliang** [babel/babel#13701]
### 💻

- [X] Would you like to work on a fix?

### How are you using Babel?

@babel/cli

### Input code

```js
console.log('123'.includes('2'))
console.log('123'.matchAll('1', '2'))
console.log('123'.replaceAll('1', '2'))
```

[code in Babel REPL](https://babeljs.io/repl/#?browsers=ios%3E%3D9.0%2Candroid%3E%3D4.4.4&build=&builtIns=usage&corejs=3.6&spec=false&loose=false&code_lz=MYewdgziA2CmB00QHMAUByAjAJgMzvgEsxhoBXAE1gg23QEp6AoUSGBJNLPAgWwEMALsAAWAQWjQMmdABoABOjqMW4KHEQppPeACdYAB2j9gsCVKxzFy5kyA&debug=false&forceAllTransforms=false&shippedProposals=false&circleciRepo=&evaluate=false&fileSize=false&timeTravel=false&sourceType=module&lineWrap=true&presets=env&prettier=false&targets=&version=7.14.4&externalPlugins=&assumptions=%7B%7D)

### Configuration file name

.babelrc

### Configuration

```json
{
    "presets": [
        [
            "@babel/preset-env", 
            {
                "targets": {
                    "browsers": ["safari >= 7", "ie >= 9"]
                },
                "useBuiltIns": "usage",
                "shippedProposals": true,
                "corejs": 3
            }
        ]
    ]
}
```

### Current and expected behavior

output:  
```js
"use strict";

require("core-js/modules/es.string.includes.js");

require("core-js/modules/esnext.string.match-all.js");

console.log('123'.includes('2'));
console.log('123'.matchAll('1', '2'));
console.log('123'.replaceAll('1', '2'));
```

why `replaceAll` not polyfilled or how to polyfill it ?

### Environment

  System:
    OS: macOS 10.15.7
  Binaries:
    Node: 14.16.1 - ~/.nvm/versions/node/v14.16.1/bin/node
    npm: 6.14.12 - ~/.nvm/versions/node/v14.16.1/bin/npm
  npmPackages:
    @babel/cli: ^7.14.8 => 7.14.8 
    @babel/core: ^7.15.0 => 7.15.0

**Comment by JLHwung** [comment]
`String#replaceAll` is supported in `core-js` `3.8.0`. Note that `3` will be interpreted as `3.0` as `preset-env` does not know what version of `core-js` you are using. Consider use a string version for `corejs`:

```json
{
    "presets": [
        [
            "@babel/preset-env", 
            {
                "corejs": "3.16"
            }
        ]
    ]
}
```
See also https://babeljs.io/docs/en/babel-preset-env#corejs.

**Comment by xwenliang** [comment]
@JLHwung Thanks for your reply !

But even changed {"corejs": 3} to {"corejs": "3.16"}, i still get the same output

Maybe we should use `es.string.replace-all` instead of `esnext.string.replace-all`:  

https://github.com/babel/babel-polyfills/blob/main/packages/babel-plugin-polyfill-corejs3/src/built-in-definitions.js#L549

**Comment by JLHwung** [comment]
> Maybe we should use es.string.replace-all instead of esnext.string.replace-all:

Yes it will be fixed in https://github.com/babel/babel-polyfills/pull/98

**Comment by nnatter** [comment]
Hey everybody,
I think we are running into the problem that `String.prototype.replaceAll` is not polyfilled when using the following configuration:

```json
{
    "presets": [
        [
            "@babel/preset-env",
            {
                "useBuiltIns": "usage",
                "corejs": "3.18",
                "targets": {
                    "node": "8"
                },
                "debug": true
            }
        ]
    ]
}
```

This example code leads to the following debug output:
```js
console.log('123'.matchAll('1', '2'))
console.log('123'.replaceAll('1', '2'))
``` 

```
corejs3: `DEBUG` option

Using targets: {
  "node": "8"
}

Using polyfills with `usage-global` method:

[/Users/niklas/Development/projects/sulu-skeleton/assets/admin/index.js]
The corejs3 polyfill added the following polyfills:
  es.string.match-all { "node":"8" }
```

Is there a workaround beside of explicitly importing the polyfill for the `String.prototype.replaceAll`?

Thanks a lot for your time and all the good work you are doing!

**Comment by nicolo-ribaudo** [comment]
I just released `babel-plugin-polyfill-corejs3@0.2.5` with the fix for this!
